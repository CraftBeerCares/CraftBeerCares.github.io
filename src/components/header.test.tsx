import { render } from "@testing-library/react";
import Header from "./header";

it("renders Header", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
