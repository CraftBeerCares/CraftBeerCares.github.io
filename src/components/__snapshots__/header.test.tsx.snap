// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header 1`] = `
<div>
  <header
    class="header"
  >
    <button
      class="burgerMenuButton"
    >
      Menu
    </button>
    <h1
      class="className title"
    >
      Craft Beer Cares
    </h1>
    <nav
      class="topNav className"
    >
      <a
        class="navLink"
        href="/"
      >
        Home
      </a>
      <a
        class="navLink"
        href="/about"
      >
        About
      </a>
      <a
        class="navLink"
        href="/join_us"
      >
        Join Us
      </a>
      <a
        class="navLink"
        href="/events"
      >
        Events
      </a>
      <a
        class="navLink"
        href="/shop"
      >
        Shop
      </a>
      <a
        class="navLink"
        href="/contact"
      >
        Contact
      </a>
    </nav>
  </header>
</div>
`;
